subprojects/plugin-development/src/test/groovy/org/gradle/plugin/devel/internal/precompiled/PreCompiledScriptTest.groovy

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.plugin.devel.internal.precompiled
+
+import org.gradle.plugin.internal.InvalidPluginIdException
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class PreCompiledScriptTest extends Specification {
```

Comment:
:o: maybe name `PreCompiledGroovyScriptTest` so it's easier to find/distinguish from Kotlin support

Reply:
Also, I don't see a good reason to deviate from the Kotlin DSL spelling of `Precompiled*` with a lower case `c` instead of `PreCompiled*` with a capital `C`. The spelling was taken from official technical documentation found at [Microsoft](https://docs.microsoft.com/en-us/cpp/build/creating-precompiled-header-files?view=vs-2019) and other sources. 

Could we use the same spelling on both implementations?

Reply:
Sure, will make sure it is aligned. Thanks!